src/components/Map/Map.tsx

```diff
@@ -27,6 +45,24 @@ export default function Map({ countries, disputedAreas }: MapProps) {
     >
       <AlertContainer countries={countries} />
       {countries && <VectorTileLayer countries={countries} disputedAreas={disputedAreas} />}
+      {selectedMapType === GlobalInsight.FOOD &&
+        countries.features &&
+        countries.features
+          .filter((countryData) => countryData.properties.interactive)
+          .filter((countryData) => countryData.properties.fcs !== null)
+          .map((countryData) => (
```

Comment:
maybe not the scope of this task, but we should use this condition on the global map to display countries as interactive or disabled. Maybe we could look into whether we can change which countries are clickable on each map, because I assume it will be different on the IPC map for example

Reply:
yeah, we could do this as a separate task, once for all map types. Here is the [link](https://linear.app/world-food-programme-wfp/issue/F-89/clickable-countries-in-global-view)